observations/README.md

```diff
@@ -0,0 +1,1279 @@
+
+<details>
+<summary>Example JSON</summary>
+
+```JSON
+{
+  "resource": {
+    "resourceType": "Observation",
+    "id": "2418B6B6-C4C0-46CB-9030-5B7DD39C80FC",
+    "meta": {
+      "profile": [
+        "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Observation-1"
+      ]
+    },
+    "identifier": [
+      {
+        "system": "https://PSSAdaptor/D5445",
+        "value": "2418B6B6-C4C0-46CB-9030-5B7DD39C80FC"
+      }
+    ],
+    "status": "final",
+    "category": [
+      {
+        "coding": [
+          {
+            "system": "http://hl7.org/fhir/observation-category",
+            "code": "laboratory",
+            "display": "Laboratory"
+          }
+        ]
+      }
+    ],
+    "code": {
+      "text": "CHOL/HDL RATIO"
+    },
+    "subject": {
+      "reference": "Patient/80bd1375-a1e3-4137-95ff-fb3316cf3496"
+    },
+    "context": {
+      "reference": "Encounter/1449860E-3953-4D71-A867-3E1E79D2E11B"
+    },
+    "effectiveDateTime": "2010-01-20T10:46:22+00:00",
+    "issued": "2010-03-26T13:49:48.000+00:00",
+    "performer": [
+      {
+        "reference": "Practitioner/1E473786-E7FA-785E-C911-A8D38FB56F20"
+      }
+    ],
+    "comment": "See FATS/Healthy Hearts guidelines for interpretation of lipids",
+    "specimen": {
+      "reference": "Specimen/92BB4158-8984-4898-8C4D-EBFD27514947"
+    },
+    "related": [
+      {
+        "type": "has-member",
+        "target": {
+          "reference": "Observation/C737A049-F93E-4C52-AFDF-21B0D1C7298C"
+        }
+      },
+      {
+        "type": "has-member",
+        "target": {
+          "reference": "Observation/E47B3A50-EEBE-4336-AA48-5932A01BC1B5"
+        }
+      },
+      {
+        "type": "has-member",
+        "target": {
+          "reference": "Observation/8673E805-9884-4040-993A-D72AECF4D363"
+        }
+      }
+    ]
+  }
+}
+```
+
+</details>
+
+### Test Result (XML HL7 > JSON FHIR)
+
+Except the related and comment fields, Test Result observations are mapped identically to [Uncategorised Data](#Uncategorised-Data-(XML-HL7->-JSON-FHIR)). 
+
+| Mapped to (JSON FHIR Observation resource field) | Mapped from (XML HL7 / other)                                                                                           |
+|--------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| comment                                          | concatenated with newlines from child `NarrativeStatement / text` where the EDIFACT comment type is not `USER COMMENT`  | 
+| related\[index].type                             | fixed value = `"derived-from"`                                                                                          |
+| related\[index].target                           | reference to the parent [Test Group Header](#Test-Group-Header-(XML-HL7->-JSON-FHIR))                                   |
```

Comment:
I _think_ this table is missing `specimen` and `category` fields.

Reply:
Added the missing fields